src/handwritten/errors.py

```diff
@@ -45,18 +45,26 @@ def __init__(self, message, error_code, task_name=''):
             message (string): Specifies the error message.
             error_code (int): Specifies the NI-DAQmx error code.
         """
-        if task_name:
-            message = f'{message}\n\nTask Name: {task_name}'
-
-        super().__init__(message)
-
         self._error_code = int(error_code)
 
         try:
             self._error_type = DAQmxErrors(self._error_code)
         except ValueError:
             self._error_type = DAQmxErrors.UNKNOWN
 
+        # If message is empty, we try to put at least some information in it
+        if not message:
+            message = f'Explanation could not be found for the requested status code.\n\nVerify that the requested status code is correct.\n\nStatus Code: {self._error_code}'
```

Comment:
The user getting the error can't verify anything, I don't think...

```suggestion
            message = f'Explanation could not be found for the status code.\n\nStatus Code: {self._error_code}'
```

Reply:
That string came from @bkeryan. Is this change acceptable?

Reply:
@zhindes It's not a new error message but your change is ok with me.

```
      /* hex: 0xFFFCF1D5 */
      /* Explanation could not be found for the requested status code.<LF/><LF/>Verify that the requested status code is correct. */
      kErrorExplanationNotFound = -200235,
```

Reply:
I think "Verify that the requested status code is correct." probably was intended to mean "Verify that you didn't pass DAQmxGetErrorString a bogus status code like 0xCCCCCCCC" but in this context, that text is not as appropriate.